frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1179,17 +1179,20 @@ function ReaderHighlight:onClose()
     self:clear()
 end
 
-function ReaderHighlight:makeDefault(highlight_disabled)
-    local new_text
-    if highlight_disabled then
-        new_text = _("Disable highlight by default.")
-    else
-        new_text = _("Enable highlight by default.")
-    end
-    UIManager:show(ConfirmBox:new{
-        text = new_text,
-        ok_callback = function()
-            G_reader_settings:saveSetting("highlight_disabled", highlight_disabled)
+function ReaderHighlight:toggleDefault()
+    local highlight_disabled = G_reader_settings:isTrue("highlight_disabled")
+    UIManager:show(MultiConfirmBox:new{
+        text = highlight_disabled and _("Enable highlighting by default.")
```

Comment:
Same basic comment as above, but I assume highlighting is straightforward enough.

```
Would you like to enable or disable highlighting by default? The current default (★) is enabled.
```